src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -61,8 +78,17 @@ export default {
 				'Oceania'
 			],
 			loadingSemaphore: 0,
+			isApolloLoading: true
 		};
 	},
+	apollo: {
+		query: pageQuery,
+		preFetch: true,
```

Comment:
Prefetch to be removed which allows us to not add the `countryFacets` to the categoryRows query. We only want this in the client.

Reply:
👍